tests/integration/framework/process/metrics/metrics.go

```diff
@@ -106,3 +106,14 @@ func (m *Metrics) MatchMetric(substrings ...string) []Metric {
 
 	return result
 }
+
+// MatchMetricAndValue returns true if the metric name AND value match the substrings
+func (m *Metrics) MatchMetricAndValue(value float64, substrings ...string) bool {
+	metrics := m.MatchMetric(substrings...)
+	matchSum := 0.0
+	for _, metric := range metrics {
```

Comment:
The sum here is not completely intuitive, from the name, but the function is simple enough to understand, so we can leave it.